nix/src/main/java/org/gnode/nix/Utils.java

```diff
@@ -0,0 +1,60 @@
+package org.gnode.nix;
```

Comment:
Can you move this to `org.gnode.nix.internal` or something similar. Just to make clear that it is not part of the public api.